### ISSUE TITLE
Accessibility mode does not change dropdowns to classic <select>

### ISSUE THREAD

**Issue by SammyIsConfused** [equella/Equella#667]
**Describe the bug**
When you select accessibility mode in the user settings, the drop downs in openEQUELLA remain as stylish select dropdowns.

**To Reproduce**
1. Log in to a non-admin user account.
2. Click on the user settings icon in the top right of the screen and select "My Preferences".
3. Set the "Accessibility mode" checkbox to checked.
4. Hit "Save".
5. Note that the dropdowns are still using stylish select.

**Expected behavior**
Accessibility mode is supposed to set the dropdowns to be just classic \<select> tags, and did so in 6.6. This change also fails the AccessibilityModeTest in the autotest suite.

**Platform:**
 - OpenEquella Version: 2018.2
 - OS: Ubuntu 18.04.1
 - Browser Firefox

**Comment by mrblippy** [comment]
Verified in 2018.2
